src/libraries/System.Net.Http/src/ILLink/ILLink.Suppressions.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<linker>
+  <assembly fullname="System.Net.Http">
+    <attribute fullname="System.Diagnostics.CodeAnalysis.UnconditionalSuppressMessageAttribute">
+      <argument>ILLink</argument>
+      <argument>IL2075</argument>
+      <property name="Scope">member</property>
+      <property name="Target">M:System.Net.Http.HttpClient.CreateDefaultHandler()</property>
```

Comment:
@EgorBo @marek-safar -  What is the plan to address this warning? We had this library ILLink "clean" already, and so it was no longer being tracked by #45623.

In the future, we should not be adding new ILLink warning suppressions to the libraries that are already clean. We need to address the ILLink warnings on any new features we add as we go.

Reply:
The suppression is intentional and will stay there. We could ideally move it to Library.xml but I think we tried that and it had some problems.